### PR TITLE
Check kibana version condition value

### PR DESCRIPTION
**Pull request by mrodm**
## What does this PR do?

Review check about the condition related to the Kibana version in package manifest.

At least there is one package (memcached) in the integrations-daily job  that is being executed with the Elastic stack 7.17.0 , but the conditions related to its [kibana version is `^8.2.0`](https://github.com/elastic/integrations/blob/028ad0851/packages/memcached/manifest.yml#L23-L25) ([Jenkins job](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations/detail/main/470/pipeline/)) 

```yaml
conditions:
  kibana:
    version: ^8.2.0
```
 
Currently, there are two ways of defining this kibana.version condition in the the manifests of the integrations/packages:
- https://github.com/elastic/integrations/blob/028ad0851/packages/1password/manifest.yml#L11-L12
- https://github.com/elastic/integrations/blob/028ad0851/packages/hid_bravura_monitor/manifest.yml#L13-L15

This PR updates the if statement to be sure to check both approaches.

### CODE REVIEW

**Comment by mrodm** [comment]
Error related to package docker should be unrelated to this change. Those errors are in main branch too.

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -260,6 +260,10 @@ def isPrAffected(integrationName) {
   // Packages supported in Kibana >= 8.0.0, shouldn't be included in daily tests of the stack 7.x
   if (manifest != null && manifest['conditions'] != null) {
     def kibanaVersionCondition = manifest['conditions']['kibana.version']
+    if (kibanaVersionCondition == null) {
+      kibanaVersionCondition = manifest['conditions']['kibana']['version']
+    }
```
Comment (jsoriano) [inline .ci/Jenkinsfile:265]: Good one :+1: 

The use of ucfg does complicate things sometimes :confused:

**Review by jsoriano — APPROVED** [review]

**Comment by mrodm** [comment]
Mergin PR since failures in CI are unrelated to this change.

**Pull request merged**